experimenter/bin/manifest-tool.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import sys
+from pathlib import Path
+
+
+if __name__ == "__main__":
+    sys.path.append(str(Path(__file__).parent.parent / "experimenter" / "features"))
+
+    from manifesttool import main
```

Comment:
Why is this necessary?

Reply:
It lets you import manifesttool directory instead of importing `experimeter.features.manifesttool`. Because the experimenter package is a Django app, there are side-effects from importing `experimenter` directly. Basically lets us have a CLI app without importing initializing all the django gunk